SpatialGDK/Documentation/toc.md

```diff
@@ -1,12 +1,16 @@
 - <h3>[- SpatialOS GDK for Unreal]({{urlRoot}}/index)</h3>
```

Comment:
The ToC doesn't play nicely - the current one isn't quite right.
It's a bit of a pain in the butt and needs tinkering with to get right. I can do this tomorrow with improbadoc. 
Merge this one in and then I will have a fiddle to make it match the others but work for itself